samtranslator/translator/translator.py

```diff
@@ -181,7 +189,7 @@ def translate(self, sam_template, parameter_values, feature_toggle=None, passthr
             if deployment_preference_collection.needs_resource_condition():  # type: ignore[no-untyped-call]
                 new_conditions = deployment_preference_collection.create_aggregate_deployment_condition()  # type: ignore[no-untyped-call]
                 if new_conditions:
-                    template.get("Conditions").update(new_conditions)
+                    template.get("Conditions", {}).update(new_conditions)
```

Comment:
Bug that mypy caught with the extra types?

Reply:
it came up when I ran `make lint` if thats what you mean

Reply:
Yeah, so it uncovered a bug. Which is nice 👍